<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание табличек</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tw_label">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <column>
       <property name="text">
        <string>Ширина</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Высота</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Шрифт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_height">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ширина</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Строка 3 (Заметка)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Строка 1 (Артикул)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_width">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_font_size">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Размер шрифта</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Высота</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Строка 2 (Имя)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Добавить товар</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="QPlainTextEdit" name="le_row1"/>
      </item>
      <item row="1" column="3" rowspan="4">
       <widget class="QPlainTextEdit" name="le_row2"/>
      </item>
      <item row="1" column="4" rowspan="4">
       <widget class="QPlainTextEdit" name="le_row3"/>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>VS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>M</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Удалить маркировку</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_product()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tw_label</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_select_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_add_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_del_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_can()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_acc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_small_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>-79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_medium_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>-39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_large_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_very_small_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>-61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_view_product()</slot>
  <slot>ui_select_label()</slot>
  <slot>ui_add_label()</slot>
  <slot>ui_del_label()</slot>
  <slot>ui_can()</slot>
  <slot>ui_acc()</slot>
  <slot>ui_small_label()</slot>
  <slot>ui_medium_label()</slot>
  <slot>ui_large_label()</slot>
  <slot>ui_very_small_label()</slot>
 </slots>
</ui>
